core/App/ehsm_napi.h

```diff
@@ -290,7 +290,7 @@ char* NAPI_GenerateQuote(const char *challenge);
  *          }
  *      }
  */
-char* NAPI_VerifyQuote(const char *quote_base64, const char *nonce);
+char* NAPI_VerifyQuote(const char *quote_base64, const char *mr_signer_base, const char *mr_enclave_base, const char *nonce_base64);
```

Comment:
same here, mrsigner/mrenclave is a single word.

Reply:
copy understood